packages/components/src/internal/schemas.ts

```diff
@@ -70,6 +70,13 @@ export const DATA_CLASSES = {
     MIXTURES: SchemaQuery.create(DATA_CLASS_SCHEMA, 'Mixtures'),
 };
 
+// INVENTORY
+const INVENTORY_SCHEMA = 'inventory';
+export const INVENTORY = {
```

Comment:
I assume this const is already defined in the FM package. Are we worried about those getting out of sync? likely no because I assume that one already has many more tables defined.